ql/src/test/org/apache/hadoop/hive/ql/exec/repl/TestAtlasDumpTask.java

```diff
@@ -242,6 +251,25 @@ public void testAtlasClientTimeouts() throws Exception {
             AtlasRestClientBuilder.ATLAS_PROPERTY_READ_TIMEOUT_IN_MS));
   }
 
+  @Test
+  public void testCreateExportRequest() throws Exception {
```

Comment:
what happens if policy is modified to remove the table expression or modify the table expression? Does it fallback to db level replication?

Reply:
Hive replication as such will not allow falling back from table level to db level replication. For a expression modification case, Atlas server should handle the case and if it does not. we will have to get that fixed by Atlas team. For now, there is no known case on Atlas server side.